kubejs/server_scripts/crafting.js

```diff
@@ -437,11 +437,165 @@ ServerEvents.recipes((event) => {
             C: "minecraft:copper_block",
         })
+    
+    event
+        .shaped("createaddition:modular_accumulator", ["NRN", "CBC", "NSN"], {
+            N: "#forge:plates/nickel",
+            R: "#forge:rods/copper",
+            C: "createaddition:capacitor",
+            B: "create:brass_casing",
+            S: "createaddition:electrum_spool",
+        })
+        .id("createaddition:crafting/modular_accumulator_electrum");
 
     //Thermal
     event.replaceInput(
         { id: "thermal:charge_bench" },
         { tag: "forge:ingots/lead" },
         "mmt:lunium_nova_plate"
     );
+    
+    //Applied Energistics
+    event
+        .shaped("ae2:meteorite_compass", [" Q ", "PGP", " P "], {
+            Q: "ae2:certus_quartz_crystal",
+            P: "#forge:plates/iron",
+            G: "#forge:gears/invar"
+        }).id("ae2:charger/meteorite_compass")
+    
+    event
+        .replaceInput({id: "ae2:network/blocks/crystal_processing_quartz_growth_accelerator"}, 
+            "#forge:ingots/iron", "#forge:ingots/silver");
+    
+    event
+        .shaped("ae2:energy_acceptor", ["TIT", "CAC", "TIT"], {
+            T: "#forge:plates/tin",
+            I: "#forge:plates/iron",
+            C: "createaddition:connector",
+            A: "createaddition:modular_accumulator"
+        })
+        .id("ae2:network/blocks/energy_energy_acceptor");
+    
+    event
+        .shaped(Item.of("ae2:formation_core", 4), ["QS ", "RFR", " SL"], {
+            Q: "create:polished_rose_quartz",
+            S: "#forge:plates/silicon",
+            R: "#forge:rods/silver",
+            L: "ae2:logic_processor",
+            F: "ae2:fluix_dust",
+        })
+        .id("ae2:materials/formationcore");
+    
+    event
+        .shaped(Item.of("ae2:annihilation_core", 4), ["LS ", "RFR", " SA"], {
+            F: "ae2:fluix_dust",
+            S: "#forge:plates/silicon",
+            R: "#forge:rods/silver",
+            L: "ae2:logic_processor",
+            A: "#forge:gems/amethyst"
+        })
+        .id("ae2:materials/annihilationcore");
+
+    event
+        .shaped("ae2:semi_dark_monitor", [" GQ", "SRQ", " GQ"], {
+            G: "minecraft:glowstone_dust",
+            Q: "ae2:quartz_glass",
+            S: "#forge:ingots/silver",
+            R: "minecraft:redstone"
+        })
+        .id("ae2:network/parts/panels_semi_dark_monitor");
+    
+    event
+        .replaceInput({id: "ae2:network/blocks/energy_energy_cell"}, 
+            Item.of("ae2:quartz_glass"), Item.of("createaddition:capacitor"));
+
+    const iron_to_silver = [
+        "ae2:material/basiccard",
+        "ae2:materials/advancedcard",
+        "ae2:network/blocks/interfaces_interface",
+        "ae2:network/crafting/cpu_crafting_unit",
+    ];
+    
+    iron_to_silver.forEach(recipe => {
+        event
+            .replaceInput({ id: recipe },
+                Item.of("minecraft:iron_ingot"), "#forge:ingots/silver");
+    });
```

Comment:
I don't really like naming things when not necessary
You can skip the const by doing either of these
I prefer option 2
```suggestion
    // iron to silver
    [
        "ae2:materials/basiccard",
        "ae2:materials/advancedcard",
        "ae2:network/blocks/interfaces_interface",
        "ae2:network/crafting/cpu_crafting_unit",
    ].forEach(recipe => {
        event
            .replaceInput({ id: recipe },
                Item.of("minecraft:iron_ingot"), "#forge:ingots/silver");
    });
```
```suggestion
    // iron to silver
    event.replaceInput(
        [
            { id: "ae2:materials/basiccard" },
            { id: "ae2:materials/advancedcard" },
            { id: "ae2:network/blocks/interfaces_interface" },
            { id: "ae2:network/crafting/cpu_crafting_unit" },
        ],
        Item.of("minecraft:iron_ingot"),
        "#forge:ingots/silicon"
    );
```